### PR TITLE
Test unit_tests/cli with queue and scheduler

### PR DESCRIPTION
**Pull request by berland**
**Issue**
Resolves bullet issue in #6810

**Approach**
_Short description of the approach_

(Screenshot of new behavior in GUI if applicable)

## Pre review checklist

- [ ] Read through the code changes carefully after finishing work
- [ ] Make sure tests pass locally (after every commit!)
- [ ] Prepare changes in small commits for more convenient review (optional)
- [ ] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Updated documentation
- [ ] Ensured that unit tests are added for all new behavior (See
    [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)),
    and changes to existing code have good test coverage.

## Pre merge checklist
- [ ] Added appropriate release note label
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: tests/unit_tests/cli/test_integration_cli.py [hunk 1]
```diff
@@ -645,7 +648,10 @@ def test_that_the_model_raises_exception_if_active_less_than_minimum_realization
 
 
 @pytest.mark.usefixtures("copy_poly_case")
-def test_that_the_model_warns_when_active_realizations_less_min_realizations():
+@pytest.mark.scheduler
+def test_that_the_model_warns_when_active_realizations_less_min_realizations(
+    monkeypatch, try_queue_and_scheduler
```
Comment (jonathan-eq) [inline tests/unit_tests/cli/test_integration_cli.py:653]: Maybe we should put the new fixtures in the mark above

**Review by jonathan-eq — CHANGES_REQUESTED** [review]

File: tests/unit_tests/cli/test_integration_cli.py (hunk 1 above)
Comment (berland) [reply tests/unit_tests/cli/test_integration_cli.py:653]: Hmm, should be its own PR if so.

**Review by xjules — APPROVED** [review]
Very nice! :rocket:

**Review by jonathan-eq — APPROVED** [review]
LGTM!

**Pull request merged**